backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -171,7 +163,26 @@ public static List<FeedbackMessage> validateTestResult(ValueOrError input) {
   }
 
   public static List<FeedbackMessage> validateSpecimenType(ValueOrError input) {
-    return validateSpecificValueOrSNOMED(input, SPECIMEN_TYPE_VALUES);
+    List<FeedbackMessage> errors = new ArrayList<>();
+    String value = parseString(input.getValue());
+
+    if (value == null) {
+      return errors;
+    }
+
+    boolean nonSNOMEDValue = value.matches(ALPHABET_REGEX);
+
+    if (nonSNOMEDValue) {
+      return errors;
+    }
```

Comment:
Would this not stop us from getting to the point where we replace the given value with our value from the snomed map?

Reply:
Nope! This line isn't actually returning any errors (although who could blame you for thinking that?) - the empty array being returned here is just saying `no errors found, validation passes ✓`.

Does that answer your question?

Reply:
gotcha, yeah that's just a little bit confusing